meson.build

```diff
@@ -0,0 +1,231 @@
+  '-Wmissing-include-dirs',
+  '-Waddress',
+  '-Waggregate-return',
+  '-Wno-multichar',
+  #'-Wdeclaration-after-statement',
+  #'-Wvla',
+  '-Wpointer-arith',
+]
+foreach extra_arg : warning_flags
+  if cc.has_argument (extra_arg)
+    # Add flag to the compiler command line
+    add_project_arguments([extra_arg], language: 'c')
+  endif
+endforeach
+
+##############################
+## Set config.h information
+gst_shark_repo = 'https://github.com/RidgeRun/gst-shark'
+cdata.set_quoted('GST_API_VERSION', apiversion, description: 'GStreamer API version')
+cdata.set_quoted('GST_SHARK_API_VERSION', f'@gst_shark_version_major@.0', description : 'GstShark API Version')
+cdata.set_quoted('GST_SHARK_LICENSE', 'LGPL', description: 'GstShark license')
+cdata.set_quoted('GST_SHARK_PACKAGE_CREATED', '2016-01-29', description: 'GstShark date of creation')
+cdata.set_quoted('GST_SHARK_PACKAGE_MAILINGLIST', '', description: 'GstShark mailing list address')
+cdata.set_quoted('GST_SHARK_README_LONG', 'GstShark leverages GStreamers newest instrumentation subsystem by installing a set of custom hooks to the trace points. A variety of data analytics are extracted and presented to the user in a graphical, friendly way.  ', description: 'GstShark long description')
+cdata.set_quoted('GST_SHARK_README_SHORT', 'A Front End for GstShark', description: 'GstShark short description')
+cdata.set_quoted('GST_SHARK_REPO_BROWSE', gst_shark_repo, description: 'GstShark read-only git repo')
+cdata.set_quoted('GST_SHARK_REPO_DOWNLOAD', '', description: 'GstShark download address')
+cdata.set_quoted('GST_SHARK_REPO_LOCATION', 'git@github.com:RidgeRun/gst-shark.git', description: 'GstShark git repo')
+cdata.set_quoted('LIBDIR', join_paths(prefix, get_option('libdir')))
+cdata.set_quoted('LOCALEDIR', join_paths(prefix, get_option('localedir')))
+cdata.set_quoted('PACKAGE_BUGREPORT', gst_shark_repo, description: 'Define to the address where bug reports for this package should be sent.')
+cdata.set_quoted('PACKAGE_NAME', 'GstShark', description: 'Define to the full name of this package.')
+cdata.set_quoted('PACKAGE_STRING', 'GstShark @0@'.format(gst_shark_version), description: 'Define to the full name and version of this package.')
+cdata.set_quoted('PACKAGE_TARNAME', 'gst-shark', description: 'gst-shark')
+cdata.set_quoted('PACKAGE_URL', gst_shark_repo, description: 'Define to the home page for this package.')
+cdata.set_quoted('PACKAGE_VERSION', gst_shark_version, description: 'Define to the version of this package.')
+cdata.set_quoted('PACKAGE', 'gst-shark', description: 'Name of package')
+cdata.set_quoted('PLUGINDIR', plugins_install_dir, description: 'Directory where plugins are located')
+cdata.set_quoted('VERSION', gst_shark_version, description: 'Version number of package')
+
+if gst_shark_version_nano > 0
+    # Have GST_ERROR message printed when running from git
+    cdata.set('GST_LEVEL_DEFAULT', 'GST_LEVEL_ERROR', description : 'Default errorlevel to use')
```

Comment:
We set the nano to `1`. Is this OK?

Reply:
Fixed this by setting the correct version